frontend/www/js/omegaup/components/RankTable.vue

```diff
@@ -97,9 +111,14 @@ export default {
     resultTotal: Number,
   },
   data: function() {
-    return { T: T, UI: UI, }
+    return { T: T, UI: UI, Searched_user: '', }
   },
   methods: {
+    onSubmit: function() {
+      var Searched_user_url = '/profile/' + this.Searched_user;
+      window.location = Searched_user_url;
```

Comment:
note that just blindly appending user-controlled input is dangerous: https://excess-xss.com/

you need to escape the input before adding it to a URL:

```suggestion
      window.location = `/profile/${encodeURIComponent(this.searchedUsername)}`;
```

You can read a lot more about why data validation is important and other ways it can go wrong at https://www.owasp.org/index.php/Data_Validation